api/libproject/devstart.rb

```diff
@@ -1764,7 +1775,7 @@ def create_project_resources(gcc)
   common = Common.new
   common.status "Enabling APIs..."
```

Comment:
Will do. (Its an existing log though, so in general out of scope. Small enough change, but I think we should tend to avoid changing out of band things within the scope of a ticket unless its a refactor.)

Reply:
It's a slippery slope, but I think small, frequent refactors when you're in the neighborhood can be really helpful.